src/raftstore/store/util.rs

```diff
@@ -31,13 +31,21 @@ use super::engine::{IterOption, Iterable};
 use super::peer_storage;
 
 pub fn find_peer(region: &metapb::Region, store_id: u64) -> Option<&metapb::Peer> {
-    for peer in region.get_peers() {
-        if peer.get_store_id() == store_id {
-            return Some(peer);
-        }
-    }
+    region
+        .get_peers()
+        .iter()
+        .find(|p| p.get_store_id() == store_id)
+}
 
-    None
+pub fn find_learner(region: &metapb::Region, store_id: u64) -> Option<&metapb::Peer> {
+    region
+        .get_learners()
+        .iter()
+        .find(|p| p.get_store_id() == store_id)
+}
+
+pub fn find_peer_or_learner(region: &metapb::Region, store_id: u64) -> Option<&metapb::Peer> {
```

Comment:
I think `find_peer` is OK. `peer` means replica, it's not the opposite of learner. The opposite of learner should be voter. Both learners and voters are peers.